netbox/extras/dashboard/widgets.py

```diff
@@ -318,9 +318,10 @@ def get_feed(self):
                 timeout=3
             )
             response.raise_for_status()
-        except requests.exceptions.RequestException as e:
+        except requests.exceptions.RequestException:
+            feed_error = 'NetBox is unable to connect to feed server...'
```

Comment:
This isn't a very helpful error message. While we can clean up the display of the exception, we do want to retain the original error message, which can offer more insight into the underlying problem. For example, "Temporary failure in name resolution" offers a much bigger clue to the problem than "NetBox is unable to connect to feed server."